app/views/articles/index.html.haml

```diff
@@ -0,0 +1,10 @@
+%h2 Articles
+
+- @articles.each do |article|
+    %img{:src => "/assets/article.jpg"}/
+    %h3= article.title
+    %p= article.body
+    %p= article.author
+    %p= article.date
+    %p= article.lede
```

Comment:
What is `lede`?

Reply:
This is [a lede](https://www.merriam-webster.com/dictionary/lede).

Reply:
Okay, thx